tributors/utils/prompt.py

```diff
@@ -31,12 +30,12 @@ def request_input():
 
 def choice_prompt(prompt, choices, choice_prefix=None, multiple=False):
     """Ask the user for a prompt, and only return when one of the requested
-       options is provided.
-       Parameters
-       ==========
-       prompt: the prompt to ask the user
-       choices: a list of choices that are valid.   
-       multiple: allow multiple responses (separated by spaces)
+    options is provided.
+    Parameters
```

Comment:
Needs empty line AFAIK
```suggestion

    Parameters
```